src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -24,18 +57,174 @@ const formConfig = {
   defaultDefinitions: {
   },
   chapters: {
-    form: {
-      title: 'Form',
+    applicantInformation: {
+      title: 'GI Bill School Complaint Tool',
       pages: {
-        page1: {
-          path: 'form-page',
-          title: 'First Page',
+        applicantInformation: {
+          path: 'applicant-information',
+          title: 'Applicant Information',
           uiSchema: {
+            onBehalfOf: {
+              'ui:widget': 'radio',
+              'ui:title': 'I’m filing on behalf of...',
+              'ui:options': {
+                nestedContent: {
+                  Myself: () => <div className="usa-alert-info no-background-image"><i>(We’ll only share your name with the school.)</i></div>,
+                  'Someone else': () => <div className="usa-alert-info no-background-image"><i>(We’ll only share your name with the school.)</i></div>,
+                  'I want to submit my complaint anonymously': () => <div className="usa-alert-info no-background-image"><i>(Your personal information won’t be shared with anyone outside of VA.)</i></div>
```

Comment:
Prefer these keys to be camel case properties but if we want to use strings they should be set as constants.

Reply:
done!